frontend/src/components/misc/ErrorBoundary.tsx

```diff
@@ -219,6 +217,29 @@ function getStringFromInfo(info: InfoItem) {
     }
 }
 
+const CopyToClipboardButton: FC<{message: string, disabled: boolean; isLoading: boolean}> = ({message, disabled, isLoading}) => {
+    const toast = useToast()
+
+    return (
+        <Button
+            variant="ghost"
+            size="large"
+            disabled={disabled}
+            isLoading={isLoading}
+            onClick={() => {
+                navigator.clipboard.writeText(message).then(() => {
```

Comment:
[question] Will this work on non-Chrome browsers? https://caniuse.com/?search=writeText

Reply:
It will as explained here https://github.com/redpanda-data/console/pull/846#discussion_r1335770588